packages/legacy/README.md

```diff
@@ -0,0 +1,94 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-legacy
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-legacy
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-legacy
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-legacy
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-legacy
+
+Add `export` declarations to legacy non-module scripts.
+
+## Installation
```

Comment:
Check out the other READMEs and how they are structured. The goal is to get every plugin's readme looking similar with the same structure so that users have little friction once they're acclimated. The opening description section, `Requirements`, `Install`, `Usage`, and `Options` sections should all exist and follow the established patterns. Even that little sushi emoji is relevant for establishing subtle branding and recognition. _(Believe me, lots of thought went into this, I promise you they aren't nitpicks)_